File: CS/GridProject/ExtendedGridView/ExtendedGridColumn.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraGrid.Columns;
using DevExpress.Utils;

namespace ExtendedGridViewSpace
{
    [System.ComponentModel.DesignerCategory("")]
    public class ExtendedGridColumn : GridColumn
    {
        public ExtendedGridColumn()
        {
            EditorShowMode = DevExpress.Utils.EditorShowMode.Default;
        }

        public EditorShowMode EditorShowMode { get; set; }
        
    }
}

File: VB/GridProject/ExtendedGridView/ExtendedGridColumn.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraGrid.Columns
Imports DevExpress.Utils

Namespace ExtendedGridViewSpace
	<System.ComponentModel.DesignerCategory("")>
	Public Class ExtendedGridColumn
		Inherits GridColumn

		Public Sub New()
			EditorShowMode = DevExpress.Utils.EditorShowMode.Default
		End Sub

		Public Property EditorShowMode() As EditorShowMode

	End Class
End Namespace
